subsys/net/lib/sockets/sockets.c

```diff
@@ -982,21 +961,32 @@ static ssize_t sock_write_vmeth(void *obj, const void *buffer, size_t count)
 static int sock_ioctl_vmeth(void *obj, unsigned int request, ...)
 {
 	switch (request) {
-	case ZFD_IOCTL_CLOSE:
-		return zsock_close_ctx(obj);
 
-	case ZFD_IOCTL_FCNTL: {
+	/* In Zephyr, fcntl() is just an alias of ioctl(). */
+	case F_GETFL:
+		if (sock_is_nonblock(obj)) {
+		    return O_NONBLOCK;
+		}
+		return 0;
```

Comment:
please follow the coding style (empty line after })

Reply:
fixed